### DISCUSSION TITLE
Colima k3d api login/password

### DISCUSSION THREAD

**Discussion by sylvainmouquet** [abiosoft/colima#751] [General]
Hello,

I get error 401 when i call the api of kubernetes : 
```
{
kind: "Status",
apiVersion: "v1",
metadata: { },
status: "Failure",
message: "Unauthorized",
reason: "Unauthorized",
code: 401
}
```

Where can i found the login/password to log in to the k8s api ?
Thanks

**Comment by abiosoft** [comment]
Which APIs precisely? Can you specify how to reproduce?

**Reply by sylvainmouquet** [reply]
the kubernetes proxy api, i will try to reproduce
